subgraph/schema.graphql

```diff
@@ -0,0 +1,73 @@
+type Token @entity {
+  id: ID!
+  creator: Creator! # address
+  name: String! # string
+  symbol: String!
+  totalSupply: BigInt!
+  decimals: Int
+  creatorTokenRatio: Int
+  isTotalSupplyFixed: Boolean
```

Comment:
All tokens created in the issuance phase are mintable, so I think `isTotalSupplyFixed` flag is unnecessary.